### ISSUE TITLE
toe ring

### ISSUE THREAD

**Issue by nicbor** [i7/counterfeit-monkey#192]
There's a bug with making a toe ring in the synthesizer from toe (tomes) and ring. The item isn't recognized for interactions.

```
>get toe ring
(I only understood the first part of that — trying anyway. Retrying that as "get toe".)
 
I can't see what you're talking about.

>get toering
I can't see what you're talking about.
```

**Comment by nicbor** [comment]
Ah this is helpful-- if anyone else runs into this bug, you can workaround by putting the hyphen 
`>get toe-ring`

**Comment by angstsmurf** [comment]
Thanks for reporting this!

It should be fixed with 23e5bc0, but there are probably other objects that have similar problems.
